src/test/java/com/vaccin/vaccin/VaccinApplicationTests.java

```diff
@@ -31,25 +29,78 @@
 
 	@Before
 	public void setUp() {
-		VaccineCenter vc = new VaccineCenter();
-		vc.setAddress("O adresa");
-		vc.setLongitude(0D);
-		vc.setLatitude(0D);
-		VaccineType vt = new VaccineType();
-		vt.setBrand("Some brand");
-		vc.setVaccineType(vt);
-		var lst = new ArrayList<>();
-		lst.add(vc);
 		Mockito.when(vaccineCenterRepository.findAll())
-				.thenReturn((List) lst);
+				.thenReturn((List) GenerateCenters.getCenters(0L,6));
+
+		Mockito.when(vaccineTypeRepository.findById(0L))
+				.thenReturn(Optional.empty());
+
+		VaccineType vt = new VaccineType();
+		vt.setId(1L);
+		vt.setBrand("Some brand 1");
+		vt.setDaysBetweenShots(60);
+		Mockito.when(vaccineTypeRepository.findById(1L))
+				.thenReturn(Optional.of(vt));
+	}
+
+	@Test
+	public void getCentersAroundCoords_GetsByRightCoords() {
+		var service = new VaccineCenterService(vaccineCenterRepository, vaccineTypeRepository);
+		assert !service.getCentersAroundCoords(0.1D,0.1D).isEmpty() : "List is empty";
+	}
+
+	@Test
+	public void getCentersAroundCoords_DoesntGetByWrong() {
+		var service = new VaccineCenterService(vaccineCenterRepository, vaccineTypeRepository);
+		assert service.getCentersAroundCoords(1D,1D).isEmpty() : "List is not empty";
+	}
+
+	@Test
+	public void addCenter_GivesAffirmativeAnswer(){
+		var service = new VaccineCenterService(vaccineCenterRepository, vaccineTypeRepository);
+		VaccineCenterCreateDto val = new VaccineCenterCreateDto();
+		val.setAddress("Some address");
+		val.setLatitude(0D);
+		val.setLongitude(0D);
+		val.setVaccineTypeId(1);
+		val.setName("Some type");
+		val.setDosesAvailable(400);
+
+		var res = service.addCenter(val);
+
+		assert res == "Vaccine Center added";
 	}
 
 	@Test
-	public void contextLoads() {
+	public void addCenter_GivesNegativeAnswer(){
 		var service = new VaccineCenterService(vaccineCenterRepository, vaccineTypeRepository);
-//		System.out.println(service.getCentersAroundCoords(0D,0D));
-		System.out.println("This is to see that the test works");
-		Assert.notEmpty(service.getCentersAroundCoords(0D,0D), "It may work");
+		VaccineCenterCreateDto val = new VaccineCenterCreateDto();
+		val.setAddress("Some address");
+		val.setLatitude(0D);
+		val.setLongitude(0D);
+		val.setVaccineTypeId(0);
+		val.setName("Some type");
+		val.setDosesAvailable(400);
+
+		var res = service.addCenter(val);
+
+		assert res == "Vaccine Type not found";
+	}
+
+	@Test
+	public void addCenter_SavesToRepo(){
+		var service = new VaccineCenterService(vaccineCenterRepository, vaccineTypeRepository);
+		VaccineCenterCreateDto val = new VaccineCenterCreateDto();
```

Comment:
Variable names could be more specific. "val" isn't even an abbreviation of VaccineCenterDto